automation/terraform/testnets/berkeley/main.tf

```diff
@@ -94,7 +94,7 @@ module "berkeley" {
   watchdog_image              = "gcr.io/o1labs-192920/watchdog:0.4.13"
   use_embedded_runtime_config = true
 
-  archive_node_count            = 3
+  archive_node_count            = 2
   mina_archive_schema           = "create_schema.sql"
   mina_archive_schema_aux_files = ["https://raw.githubusercontent.com/MinaProtocol/mina/8468f29424fa5e70cfedf69ee281f4fa24037dfe/src/app/archive/create_schema.sql", "https://raw.githubusercontent.com/MinaProtocol/mina/8468f29424fa5e70cfedf69ee281f4fa24037dfe/src/app/archive/zkapp_tables.sql"]
```

Comment:
I know that it wasn't your change but I'm a bit worried to have hardcoded path to commit. Do you have plan to use some placeholder ? or fail if we won't override those variables?